marxan.cpp

```diff
@@ -130,8 +130,9 @@ namespace marxan {
 
         printf("Running multithreaded over number of threads: %d\n", maxThreads);
         displayProgress1("Running multithreaded over number of threads: " + to_string(maxThreads) + "\n");
-#pragma omp parallel for
-        for (int i = 1; i <= repeats; i++)
+
+    #pragma omp parallel for schedule(dynamic)

```

Comment:
does this change have any noticeable runtime impact? 
My understanding is that this will print runs as they finish (rather than as a batch like before)

Reply:
Original will run runs 1,51,151,201,251 .. then 2, 52, 152, 202, 252... with loose order (51, 1, 201 ..)...  
dynamic  run runs  starting 1-12  then 12-24 and so on,  (order also loose, but much easier to see the overall progress).
Both run in perception in "batches", just batch configuration are different.